### PR TITLE
Paje/fix docs

### LINKED ISSUE #595: Documentation Icons broken
In the documentation are all Icons broken. Maybe some want to look at this.

For example:

image

Also the flowchart is broken: https://t-systems-mms.github.io/phonebook/pages/development-guides/backend/getting-started

### PR DESCRIPTION
**Pull request by paule96**
this fixes #595

And all mermaid diagrams are displayed :)

the result looks like this:

https://paule96.github.io/trygithubpages/#trygithubpages

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -89,6 +89,19 @@ <h2 class="text-delta">Table of contents</h2>
       </div>
     </div>
   </div>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.5.0/mermaid.min.js"></script>
```
Comment (paule96) [inline docs/_layouts/default.html:92]: should I put this in our local assets folder?

Comment (DanielHabenicht) [reply docs/_layouts/default.html:92]: mhm, its just the docu, so I don't think so.

Comment (DanielHabenicht) [reply docs/_layouts/default.html:92]: but maybe it's better (data privacy wise)

Comment (paule96) [reply docs/_layouts/default.html]: okay done 👍

**Pull request merged**